lib/ansible/modules/cloud/digital_ocean/digital_ocean_sshkey_info.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+
+DOCUMENTATION = '''
+---
+module: digital_ocean_sshkey_info
+short_description: Gather information about DigitalOcean SSH keys
+description:
+  - This module can be used to gather information about DigitalOcean SSH keys.
+  - This module was called C(digital_ocean_sshkey_facts) before Ansible 2.9. The usage did not change.
```

Comment:
```suggestion
```
That's not true. Better remove this, or write something like "This module replaces the C(digital_ocean_sshkey_facts) module."

Reply:
Good catch!